### ISSUE TITLE
Request for further information

### ISSUE THREAD

**Issue by TheMrguiller** [s-nlp/pseudoparadetox#1]
Hi s-nlp,

I really enjoyed your article about generating pseudoparadetox data using uncensored models. I liked it so much that, as I am currently working on something very similar, I would like to cite your work and compare it with mine. I am also comparing it with other amazing works, such as "DetoxLLM: A Framework for Detoxification with Explanations". Since I plan to use your work, I have several questions:

- One of the outcomes of your article is the generation of version 2.0 of your previous BART model for the task of paraphrasing. Do you plan to share the model that achieved the best results in your experiments?

- You compare the uncensored models’ performance on the task with the previous BART model, but do you have a comparison with BART 2.0? It would be interesting to know if, in your private test set, the BART 2.0 model significantly outperformed the uncensored models.

- In your code, you provide two uncensored models, but you do not specify which ones were used in your experiments. Are they just test models that we can use? It would be helpful to know, as I think it may also be suitable to compare your prompt and model approach for the task of paraphrasing toxic sentences in my work.

Thank you so much for your incredible work!

**Comment by etomoscow** [comment]
Hi, @TheMrguiller 

sorry for long time without responding. We have uploaded several datasets using those BART has achieved best automatic detoxification scores: 
1.https://huggingface.co/datasets/s-nlp/pseudoparadetox_llama3_70b_0shot_noabl
2.https://huggingface.co/datasets/s-nlp/pseudoparadetox_llama3_70b_10shot_noabl

---

> One of the outcomes of your article is the generation of version 2.0 of your previous BART model for the task of paraphrasing. Do you plan to share the model that achieved the best results in your experiments?

Regarding releasing the "2.0 BART" model, we're working on it and will update the response once it will be released. 

> You compare the uncensored models’ performance on the task with the previous BART model, but do you have a comparison with BART 2.0? It would be interesting to know if, in your private test set, the BART 2.0 model significantly outperformed the uncensored models.

Let me clarify the experimental setup of our paper for you once again. In this work we used LLMs for generating pseudo-parallel detoxification data, then we trained BART on this data and compared this BART to the [BART](https://huggingface.co/s-nlp/bart-base-detox) by Logacheva et al. [1]

Additionally, for reference purposes, we have tested LLama 3 models on ParaDetox test set to check their performance. Since the metrics are unified, you can compare results from Tables 2 and Table 7 freely. 


> In your code, you provide two uncensored models, but you do not specify which ones were used in your experiments. Are they just test models that we can use? It would be helpful to know, as I think it may also be suitable to compare your prompt and model approach for the task of paraphrasing toxic sentences in my work.

You can use any of the abliterated models available, e.g. [2, 3]. Ours will be released soon.


**References**
[1] Logacheva, Varvara, et al. "Paradetox: Detoxification with parallel data." Proceedings of the 60th Annual Meeting of the Association for Computational Linguistics (Volume 1: Long Papers). 2022.
[2] https://huggingface.co/mlabonne/Hermes-3-Llama-3.1-70B-lorablated
[3] https://huggingface.co/mlabonne/Hermes-3-Llama-3.1-8B-lorablated

**Comment by etomoscow** [comment]
@TheMrguiller, hello once again. We have uploaded some of the PseudoParaDetox samples on HF:

https://huggingface.co/collections/s-nlp/pseudoparadetox-67975336b72b1721ea3d69ab

You can fine-tune BART models using the configs we release:

https://github.com/s-nlp/pseudoparadetox/blob/main/configs/bart_real_train.json

Just make sure to update paths to data accordingly.
